src/Tribe/Tickets_Handler.php

```diff
@@ -1044,7 +1053,7 @@ public function save_global_stock( $post_id ) {
 	 *
 	 */
 	public function save_tickets_order( $post_id ) {
-		if ( ! ( isset( $_POST[ 'tribe-tickets-post-settings' ] ) && wp_verify_nonce( $_POST[ 'tribe-tickets-post-settings' ], 'tribe-tickets-meta-box' ) ) ) {
```

Comment:
Good point. This nonce is for the settings form - so while it is set, it's not the appropriate one. Hm.